src/LoginRouter.js

```diff
@@ -198,6 +235,18 @@ function (BaseLoginRouter, PrimaryAuthController, VerifyDuoController, MfaVerify
       });
     },
 
+    enrollWindowsHello: function () {
+      this.render(EnrollWindowsHelloController, {
+        provider: 'FIDO',
+        factorType: 'webauthn',
+        Beacon: FactorBeacon
+      });
+    },
+
+    enrollmentWindowsHello: function () {
+      this.controller.model.activate();
```

Comment:
Moved inside the `EnrollWindowsHelloController` inside the `trapAuthResponse` function.